### PR TITLE
Server room photosphere, 40th page update, and bug fixes

### PR DESCRIPTION
**Pull request by stevenmirabito**
Changes included in this PR:
- Added @rossdylan's server room photosphere to the tour page
- Added a dedicated hotel reservation information section to the 40th page
- Added the new flat panel styles to the 40th page
- Fixed a bug with the JS nav highlighting on the 40th page
- Removed an asset and page that are no longer used
- Fixed a bug with the buttons on the Insights page

### CODE REVIEW

File: 40th.html [hunk 1]
```diff
@@ -54,7 +54,24 @@ <h3 class="panel-title">Event Calendar</h3>
...
+		  <div class="panel-heading">
+		    <h3 class="panel-title">Hotel Reservations</h3>
+		  </div>
+		  <div class="panel-body">
+		  	<div class="row">
+		  		<div class="col-md-2">
+		  			<img src="resources/images/40th/hampton.png" alt="Hampton by Hilton" width="100%">
+		  		</div>
+		  		<div class="col-md-10">
+		  			<p>CSH has reserved a block of rooms at the Hampton Inn & Suites Rochester/Henrietta for the event. The hotel has created a reservation portal for members to reserve rooms in our hotel block at a special rate. Rooms start at $99/night until March 15th or until the block is sold-out, whichever comes first.</p>
```
Comment (liam-middlebrook) [inline 40th.html]: Can you switch out `&` for `&amp;`?

Comment (stevenmirabito) [inline 40th.html]: Fixed in https://github.com/stevenmirabito/CSHPublicSite/commit/dd058014bb224d92f55032c314bdec46cb46245a

**Comment by stevenmirabito** [comment]
Added a workaround for the carousel bug discussed here: https://github.com/angular-ui/bootstrap/issues/1513

File: resources/js/threejs/CanvasRenderer.js [hunk 2]
```diff
@@ -0,0 +1,1114 @@
+/**
```
Comment (sman591) [inline resources/js/threejs/CanvasRenderer.js]: This, along with the others, should probably be minified

**Comment by liam-middlebrook** [comment]
Could you break this up into separate smaller pull-requests? If they were split-up per feature then it would be easier to merge some of them faster while you fix issues with the others.

**Pull request closed without merging**